application/templates/dashboards/index.html

```diff
@@ -34,6 +34,9 @@
                     <li><a class='btn btn-success' href="{{ url_for('dashboard_clone', uuid=dashboard.id) }}">Clone</a></li>
                   {% endif %}
                   <li><a class='btn btn-primary' href="{{ dashboard['public-url'] }}" target="_blank">Preview</a></li>
+                  {% if dashboard.status == "unpublished" %}
```

Comment:
This should also test for the `big-edit` feature, so that only internal users can delete dashboards.